src/Sarif/FilteringStrategies.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public static class FilteringStrategies
+    {
+        public static Func<Result, bool> NewOrUpdatedResults =
+            (result) =>
+            {
+                return result.BaselineState == BaselineState.New ||
+                       result.BaselineState == BaselineState.Updated;
```

Comment:
>BaselineState == BaselineState.Updated [](start = 30, length = 38)

I wonder why we decided to start with new and updated rather then just new. i know this is just one of the strategies - just curious - would an updation in the code warrant a new bug?

Reply:
Agreed, let's keep this entirely focused on what's actually in play.

---
In reply to: [308474361](https://github.com/microsoft/sarif-sdk/pull/1596#discussion_r308474361) [](ancestors = 308474361)